src/labCommands.ts

```diff
@@ -265,10 +262,8 @@ export function addNotebookCommands(
           content.activeCellIndex = current.content.widgets.length - 1;
           content.deselectAll();
           if (content.activeCell !== null) {
-            ElementExt.scrollIntoViewIfNeeded(
-              content.node,
-              content.activeCell.node
-            );
+            // note: using `scrollToItem` because `scrollToCell` changes mode (activate the cell)
+            content.scrollToItem(content.activeCellIndex, 'smart');
```

Comment:
One option to fix the cell not being properly selected. Seems a bit silly, but i tried using `content.select` after scrollToItem and it didn't work

```suggestion
            content.scrollToCell(content.activeCell, 'smart').then((value) => {
                commands.execute('notebook:enter-command-mode');
              }
            );
```

Reply:
Thank you @ianhi! I pushed [be7c017](https://github.com/jupyterlab-contrib/jupyterlab-vim/pull/137/commits/be7c0178ccbb0c4355c917eff325850a54c8d50b) which solves this problem by focusing the cell explicitly. This is faster than using `scrollToCell` and executing a command because `scrollToCell` will take extra time to change mode, and executing the command will be delayed by one idle cycle which can lead to visual jitter.

Reply:
that looks awesome thank you!